dev_docs/key_concepts/api_authorization.mdx

```diff
@@ -0,0 +1,306 @@
+---
+id: kibDevDocsSecurityAPIAuthorization
+slug: /kibana-dev-docs/key-concepts/security-api-authorization
+title: Kibana API authorization
+description: This guide provides an overview of API authorization in Kibana.
+date: 2024-10-04
+tags: ['kibana', 'dev', 'contributor', 'security']
+---
+
+Security is everyone's responsibility. This is inclusive of design, product, and engineering. The purpose of this guide is to give a high-level overview of security constructs and expectations.
+
+This guide specifically focuses on API authorization:
+- [[Deprecated] Adding API Authorization with `access` Tags](#deprecated-adding-api-authorization-with-access-tags)
+- [Configuring Authorization on Routes](#configuring-authorization-on-routes)
+- [Opting Out of Authorization for Specific Routes](#opting-out-of-authorization-for-specific-routes)
+- [Classic Router Security Configuration Examples](#classic-router-security-configuration-examples)
+- [Versioned Router Security Configuration Examples](#versioned-router-security-configuration-examples)
+- [Authorization Response in Route Handlers](#authorization-response-available-in-route-handlers)
+- [OpenAPI Specification (OAS) documentation](#openapi-specification-oas-documentation)
+- [Migrating from `access` Tags to `security` Configuration](#migrating-from-access-tags-to-security-configuration)
+
+## API authorization
+Kibana API routes do not have any authorization checks applied by default. This means that your APIs are accessible to anyone with valid credentials, regardless of their permissions. This includes users with no roles assigned.
+This on its own is insufficient, and care must be taken to ensure that only authorized users can invoke your endpoints.
+
+Kibana leverages <DocLink id="kibDevDocsSavedObjectsIntro" text="Saved Objects" /> for a majority of its persistence. The Saved Objects Service performs its own authorization checks, so if your API route is primarily a CRUD interface to Saved Objects, then your authorization needs are already met.
+This is also true for derivatives of the Saved Objects Service, such as the Alerting and Cases services.
+
+If your endpoint is not a CRUD interface to Saved Objects, then your route ensure that only authorized users can invoke your endpoint. This is **especially** important if your route does any of the following:
```

Comment:
I know this was copy/paste from the old docs, but I thought this was worth clarifying.

```suggestion
If your endpoint is not a CRUD interface to Saved Objects, or if your endpoint bypasses our built-in Saved Objects authorization checks, then you must ensure that only authorized users can invoke your endpoint. This is **especially** important if your route does any of the following:
```